packages/core/src/accordion/accordion.stories.ts

```diff
@@ -244,23 +244,25 @@ export function customStyles() {
     <style>
       .app-custom {
         --border-color: midnightblue;
-        --header-color: aliceblue;
-        --header-font-weight: bold;
-        --header-background: LightSkyBlue;
-        --header-open-background: CornflowerBlue;
+        --border-radius: 0;
+        --border-width: 2px;
         --icon-color: midnightblue;
+
+        --cds-alias-object-interaction-background-selected: lightskyblue;
+        --cds-alias-object-interaction-background-hover: lightskyblue;
+        --cds-alias-object-interaction-background-active: cornflowerblue;
```

Comment:
As discussed, we'll use the interaction tokens to demo how one can customize the header color in this example. Alternatively the user can do something like this to achieve the same results:

```
.app-custom {
        --border-color: midnightblue;
        --border-radius: 0;
        --border-width: 2px;
        --icon-color: midnightblue;
      }
      .app-custom cds-accordion-panel[expanded] cds-accordion-header {
        --background: lightskyblue;
      }
      .app-custom cds-accordion-panel cds-accordion-header:hover {
        --background: lightskyblue;
      }
      .app-custom cds-accordion-panel cds-accordion-header:active {
        --background: cornflowerblue;
      }
}      
```
We have a pending task to document how to use our design tokens to theme components systematically and our recommendations for customizing components based on use cases/scenarios.